### ISSUE TITLE
Why did you use sigmoid in classification head?

### ISSUE THREAD

**Issue by samra-irshad** [yhenon/pytorch-retinanet#238]
I think sigmoid is not an appropriate choice to use in case of multiple classes in classification branch

**Comment by Coobiw** [comment]
In the paper, authors have said that they use sigmoid function to do the classification. They have tried softmax, but sigmoid is better probably.
